src/test/java/fr/paris/lutece/portal/service/daemon/DaemonSchedulerTest.java

```diff
@@ -240,26 +245,27 @@ private void testEnqueueFull( boolean shouldThrow ) throws ClassNotFoundExceptio
             inqueueDaemon.go( );
             inqueueDaemon.waitForCompletion( );
         }
+        catch( InterruptedException | BrokenBarrierException | TimeoutException e )
+        {
+            fail("failed" + e + ":" + e.getMessage( ) );  
```

Comment:
here you swallow the exception and you loose the stack trace which might be helpful. Why not let tests throw exceptions, which makes them fail ?